src/ddscxx/src/org/eclipse/cyclonedds/core/cond/StatusConditionDelegate.cpp

```diff
@@ -53,7 +53,15 @@ org::eclipse::cyclonedds::core::cond::StatusConditionDelegate::close()
 {
     /* todo: cannot close this->ddsc_entity because this is the parent EntityDelegate;
```

Comment:
I am quite surpirsed that a `StatusCondition` can be closed at all. In the spec each Entity *has* a StatusCondition, so what could be the possible meaning of closing it? Maybe I misremembering or overlooking something in the spec, if so, perhaps @e-hndrks will correct me.

Reply:
@eboasson I am also confused with this and wondering what is the correct behavior. [DDS C++ PSM](https://www.omg.org/spec/DDS-PSM-Cxx/1.0/PDF) section 7.5.1.2 says "Users of this PSM are recommended to call close on objects of all reference types once they are finished using them", at least from this it might be possible to close even status condition, or ..?

@e-hndrks